OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -183,8 +183,8 @@ public Actor ClosestProc(Actor self, Actor ignore)
 
 					var occupancy = refs[loc].Occupancy;
 
-					// 4 harvesters clogs up the refinery's delivery location:
-					if (occupancy >= 3)
+					// 3 harvesters clogs up the refinery's delivery location:
+					if (occupancy >= 2)
```

Comment:
Could be able ... this Pr was mainly the part of "Use all Refinerys in your base"  or "search another nearest free Refinery in your Base" so the value 2 allows at maximum 2 harvesters waiting at a refinery, as far i found out...